packages/components/src/composite/index.tsx

```diff
@@ -88,8 +84,6 @@ export const Composite = Object.assign(
 			rtl,
 		} );
 
-		const store = storeProp || newStore;
```

Comment:
This is the main (only) runtime change — we don't accept a `store` prop anymore on `Composite`